OSSMDOC-395: Configure Federation for failover. (#41725)
openshift/openshift-docs: modules/ossm-federation-config-destinationrule-failover.adoc

////
This module included in the following assemblies:
* service_mesh/v2x/ossm-federation.adoc
////
:_content-type: PROCEDURE
[id="ossm-federation-config-destinationrule-failover_{context}"]
= Configuring a DestinationRule for failover

Create a `DestinationRule` resource that configures the following:

* Outlier detection for the service. This is required in order for failover to function properly. In particular, it configures the sidecar proxies to know when endpoints for a service are unhealthy, eventually triggering a failover to the next locality.

* Failover policy between regions. This ensures that failover beyond a region boundary will behave predictably.

.Procedure

> . Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Then, enter your username and password when prompted.

Reviewer: ```suggestion
. Log in to the {product-title} CLI with `cluster-admin` permissions. Enter your username and password when prompted.
```
I tried to simplify this a little. Definitely not required.
